test/kanbanize_test.rb

```diff
@@ -46,7 +46,7 @@ def test_push_with_issue_tracking
     @stubs.post url do |env|
       assert_equal 'testdomain.kanbanize.com', env[:url].host
       assert_equal '/index.php/api/kanbanize/git_hub_event', env[:url].request_uri
-      assert_equal %({"ref":"refs/heads/mybranch2"}), env[:body]
```

Comment:
Should you be testing both of these in a separate test? One for the `push` event and one for the `issue` event?